posthog/tasks/usage_report.py

```diff
@@ -927,9 +927,17 @@ def get_teams_with_ai_event_count_in_period(
     return results
 
 
+dwh_pricing_free_period_start = datetime(2025, 10, 29, 0, 0, 0, tzinfo=UTC)
+dwh_pricing_free_period_end = datetime(2025, 11, 6, 8, 0, 0, tzinfo=UTC)
+
+
 @timed_log()
 @retry(tries=QUERY_RETRIES, delay=QUERY_RETRY_DELAY, backoff=QUERY_RETRY_BACKOFF)
 def get_teams_with_rows_synced_in_period(begin: datetime, end: datetime) -> list:
+    if begin >= dwh_pricing_free_period_start and begin < dwh_pricing_free_period_end:
```

Comment:
This seems inconsistent - all rows from 2025-11-06 will be excluded from this query based on how it's defined, even though `dwh_pricing_free_period_end` is set to 8am. 

Same issue applies to free historical rows query.

Should we just set `dwh_pricing_free_period_end` to midnight UTC too for simplicity?